docs/deployment.rst

```diff
@@ -1,6 +1,47 @@
 E4S Deployments
 ===============
 
+Arcticus
+----------
+
+**ParaTools Deployment: E4S 22.05**

```

Comment:
I think we should have some sentence about how one can access Arcticus and where this system is. As a end-user who has no knowledge we need a few more sentences. I dont think we have any public facing docs on this but we can link to the wiki page https://wiki.jlse.anl.gov/. I dont know the main page for arcticus. 

IT would be helpful to have link on how one can get account access to the system

Reply:
Are you sure? This is not supposed to be a documentation of Arcticus as a system. Furthermore, Arcticus is a restricted, NDA-only system. So actually, now that I think of it, I'm not sure we can say much about it.

Reply:
After discussion with @sameershende it sounds like we should refer to this deployment simply as JLSE and point users to the JLSE website: https://www.jlse.anl.gov, as Arcticus is NDA-only.